roles/prometheus/tasks/main.yml

```diff
@@ -1,9 +1,14 @@
 # code: language=ansible
 ---
+- name: Gather the version of SLES_SAP
```

Comment:
Curious that this value was available in the pre_tasks but here we need to gather it

Reply:
This is a left over, I later realized that `gather_facts` is enabled by default. I'll remove it.